nearest.go

```diff
@@ -0,0 +1,50 @@
+package osrm
+
+import geo "github.com/paulmach/go.geo"
+
+// NearestRequest represents a request to the nearest method
+type NearestRequest struct {
+	Profile  string
+	GeoPath  GeoPath
+	Bearings []Bearing
+	Number   int
+}
+
+// NearestResponse represents a response from the nearest method
+type NearestResponse struct {
+	Waypoints []NearestWaypoint `json:"waypoints"`
+}
+
+// NearestWaypoint represents a nearest point on a nearest query
+type NearestWaypoint struct {
```

Comment:
In the original doc, a waypoint response contains `nodes` (Array of OpenStreetMap node ids.):
```
{
   "waypoints" : [
      {
         "nodes": [
            2264199819,
            0
         ],
         "hint" : "KSoKADRYroqUBAEAEAAAABkAAAAGAAAAAAAAABhnCQCLtwAA_0vMAKlYIQM8TMwArVghAwEAAQH1a66g",
         "distance" : 4.152629,
         "name" : "Friedrichstraße",
         "location" : [
            13.388799,
            52.517033
         ]
      },
...
```
Could you add it as:
```
Nodes []uint64 `json:"nodes"`
```